src/Utils/sortpermFast.jl

```diff
@@ -3,15 +3,11 @@ export sortpermFast
 
 
 function sortpermFast(A)
-	const n = length(A)
-   ii = Array(Int64,n)
-
-   for i = 1:n
-   	ii[i] = i
-   end
+   const n = length(A)
 
+   ii = collect(1:n)
    B = copy(A)
-   quicksort!(B,ii)
+   quicksort!(B,ii, 1,n)
```

Comment:
I just ran some tests and surprisingly Roman's version is significantly faster for large vectors than the native version...